File: Data Structures & Algorithms/climbing-stairs/submission-0.java
class Solution {
    public int climbStairs(int n) {
        if (n==0)
        {
            return 1;
        }else if(n<0)
        {
            return 0;
        }
        int a = climbStairs(n-1);
        int b = climbStairs(n-2);
        return a + b;
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        left = 0
        right = left +1
        maxp = 0
        while right < len(prices):
            if prices[left] < prices[right]:
                maxp = max([maxp,prices[right] - prices[left]])
            else:
                left = right
            right = right + 1
        return maxp

